invenio_github/api.py

```diff
@@ -538,6 +538,17 @@ def contributors(self):
                 f"Github returned unexpected code: {status} for release {self.repository_object.github_id}"
             )
 
+    @cached_property
+    def owner(self):
+        """Get owner of repository as a creator."""
+        try:
+            owner = self.gh.api.repository_with_id(
```

Comment:
`self.gh.api.repository_with_id` is a `GET` request to GitHub. We have two cached properties (`contributors` and `owner`) that use the same object.

Maybe we can cache `repository_with_id` instead and avoid a possible extra `GET` request?

Reply:
I think the `github3.py` library also tries to do some caching, so we might be doing extra work (has to be checked with the library).